http-link-header/index.d.ts

```diff
@@ -0,0 +1,19 @@
+// Type definitions for http-link-header v0.6.0
+// Project: https://github.com/jhermsmeier/node-http-link-header
+// Definitions by: Christian Rackerseder <https://www.echooff.de/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface Reference {
+    uri: string;
+    rel: string;
+    title?: string;
+}
+export interface Link {
+    refs: Reference[];
+    has: (attribute: string, value: string) => boolean;
```

Comment:
`has(attribute: string, value: string): boolean` is an invalid syntax because it is a function signature.